lib/jsonapi/view.ex

```diff
@@ -142,15 +142,19 @@ defmodule JSONAPI.View do
       end
 
       if Code.ensure_loaded?(Phoenix) do
+        def render("show.json", %{data: data, conn: conn, params: params, meta: meta}),
+          do: show(data, conn, params, meta: meta)
+        def render("show.json", %{data: data, conn: conn, meta: meta}),
```

Comment:
One unrelated curiosity: 

If I define a render function in my controller, how does the `conn` parameter get in there?

```
render(conn, "index.json", %{data: my_list, meta: meta})
```

Reply:
conn is always merged into the assigns data, because the view doesn't mutate/return a conn its treated as data instead of the subject of the function.